src/App.css

```diff
@@ -0,0 +1,38 @@
+.App {
```

Comment:
Are we using these styles in our code ?, if not then we can delete this file safely.